wiki/fr/fr-Installation-for-Windows.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: wiki
+title: "Installation sous Windows"
+lang: "fr"
+permalink: "/wiki/Installation-for-Windows"
+---
+
+# Installation sous Windows
+
+Assurez-vous d'avoir déjà lu la page [Premier pas](Getting-Started) avant de commencer.
+
+1. **Téléchargez et installez un pilote ASIO**. Il est recommandé d'utiliser une interface audio ou une carte son avec un pilote ASIO natif. Si vous n'en avez pas (en particulier pour les cartes son internes), installez ce [pilote ASIO gratuit (ASIO4All)](http://www.asio4all.org){: target="_blank" rel="noopener noreferrer"} avant d'installer Jamulus.
+1. **Téléchargez et installez Jamulus** depuis [la page du projet Jamulus](https://sourceforge.net/projects/llcon/files/latest/download){: target="_blank" rel="noopener noreferrer"}. Si vous avez un avertissement, cliquez sur « Plus d'infos » et « Exécuter quand même » pour démarrer l'installation Jamulus.
+1. **Configurez votre interface audio**. Lorsque vous en aurrez terminé, vous devrez configurer votre matériel audio. Regardez comment configurer ASIO4All si vous l'utilisez (défilez vers le bas), et ensuite consultez la section [Configuration du matériel](Hardware-Setup).
+
+Vous pouvez également [compiler vous même le logiciel](Compiling) si vous le souhaitez.
+
+***
+
+## Configuration de ASIO4All
+*ASIO4All n'est pas recommandé si votre interface audio fournit son propre pilote ASIO, car celui-ci pourrait avoir une meilleure latence.*
+
+Tout d'abord, **fermez toutes les applications** (en particulier celles qui pourraient accéder à votre carte son comme votre navigateur) pour minimiser les conflits. Si l'audio ne fonctionne pas « sortie de la boîte », assurez-vous que seules les **entrées/sorties correctes** de son panneau de contrôle **sont activées**.  
+Vous devrez peut-être expérimenter un peu pour trouver les bonnes, car chaque ordinateur est un peu différent. Faites-le pendant que vous êtes connecté à un serveur pour entendre votre instrument ou votre voix afin de vérifier que tout soit correctement configuré.
+
+### Comment configurer les entrées ASIO4All (Guide)
+
+1. Ouvrez les paramètres de Jamulus
+1. Allez à « _ASIO Settings_ » (Configuration ASIO)
+1. Activer le panneau _advanced view_ dans ASIO4All (cliquez sur l'icône en bas à droite)
+1. Ouvrez les entrées de votre interface audio en cliquant sur l'_icône plus_ à côté du nom de l'interface.
```

Comment:
* "_interface audio_" -> "_carte son_" to match the English wording
* same for "_soundcard_" (the last word)

Reply:
English should be updated to audio interface. ;) (I note for the changes branch)

Reply:
Ouvrez les entrées de votre carte son en cliquant sur l'_icône plus_ à côté de son nom 

To avoid repetition, yes, no ?

Reply:
Yes, good thinking.

Reply:
OK, I update.